test/e2e/encryption_at_rest_test.go

```diff
@@ -302,6 +306,121 @@ var _ = Describe("Encryption at rest AWS", Label("encryption-at-rest"), func() {
 			actions.WaitForConditionsToBecomeTrue(userData, status.EncryptionAtRestReadyType, status.ReadyType)
 		})
 	})
+
+	It("Should be able to create Encryption at REST on AWS with data from the Secret", func() {
```

Comment:
I wonder if changes being added by @roothorp may also be affected by this, I mean using secrets rather than config directly in the project CR.

Reply:
It shouldn't, as these fields are not mandatory. The test itself should pass after merging @roothorp's PR